draft-ietf-jsonpath-base.md

```diff
@@ -1047,7 +1045,7 @@ relation-expr = comp-expr /                           ; comparison test
                 regex-expr                            ; regular expression test
 ~~~~
 
-Comparisons are restricted to Singular Path values and primitive values (that is, numbers, strings, `true`, `false`,
+Comparisons are restricted to Singular Path values and primitive values (that is, numbers, strings, booleans,
```

Comment:
Same here about the booleans.

Also, do we not want to include `null` for some reason?

Reply:
`null` is mentioned on the next line.

Reply:
The JSON spec mentions "booleans" [sic!] only in the introduction.  The actual spec uses `false` and `true` (e.g., Section 3).  I slightly prefer to follow this precedent, as there is nothing in JSON that makes two out of the three special values a "type" (JSON actually doesn't have types -- that word is mentioned again in the intro only, and in Section 5 where the word is used to say that JSON arrays do not use the concept).

Reply:
Ok, I'll revert those two changes.